src/contracts/core/DelegationManager.sol

```diff
@@ -288,6 +288,32 @@ contract DelegationManager is Initializable, OwnableUpgradeable, Pausable, Deleg
         return withdrawalRoots;
     }
 
+    // onlyWhenNotPaused probably needs to touch both enter/exit because this method both:
+    // - removes a pending withdrawal
+    // - adds a new pending withdrawal
+    function changeWithdrawer(
+        Withdrawal memory withdrawal,
+        address newDestination
+    ) public onlyWhenNotPaused(PAUSED_ENTER_WITHDRAWAL_QUEUE) onlyWhenNotPaused(PAUSED_EXIT_WITHDRAWAL_QUEUE) returns (bytes32) {
+        require(newDestination != address(0), "changeWithdrawer: must provide valid withdrawal address");
+        
+        // Remove the old withdrawal from the queue
+        bytes32 oldWithdrawalRoot = calculateWithdrawalRoot(withdrawal);
+        require(pendingWithdrawals[oldWithdrawalRoot], "changeWithdrawer: withdrawal is not in queue");
+
+        // Update the withdrawer address. All other fields are unchanged
+        withdrawal.withdrawer = newDestination;
+        bytes32 newWithdrawalRoot = calculateWithdrawalRoot(withdrawal);
+        require(!pendingWithdrawals[newWithdrawalRoot], "changeWithdrawer: did not modify withdrawal");
+
+        // Replace the withdrawal in state
+        delete pendingWithdrawals[oldWithdrawalRoot];
+        pendingWithdrawals[newWithdrawalRoot] = true;
+        // TODO emit some event here?
```

Comment:
definitely need to decide on events. I think we need one for both the old and the new withdrawal, or else indexing will be thoroughly messed up.
perhaps `WithdrawerChanged(withdrawal oldWithdrawal, withdrawal newWithdrawal)` or something?